src-tauri/src/algorithms/articulation_point.rs

```diff
@@ -1,3 +1,6 @@
+#![allow(dead_code)]
+#![allow(non_snake_case)]
```

Comment:
Nit: I feel like it would be worth it to keep this case check, what is causing those warnings?

Reply:
Personally I hate having unnecessary warnings in my code. My class function names are snake case and algo function names are camel case (functions that don't belong to a class).

Reply:
No actually I just name my functions randomly lol